server/src/utils/jwt.ts

```diff
@@ -0,0 +1,12 @@
+import * as jwt from 'jsonwebtoken';
+import { env } from 'src/config/env';
+
+export class Jwt {
+  public sign(userId: string): string {
```

Comment:
Tutaj dałbym zamiast userId po prostu obiekt, bo nie będziesz wykorzystywał tego serwisu tylko do autoryzacji

Reply:
Masz na myśli coś takiego?
```suggestion
  public sign(config: Object): string {
      return jwt.sign(config, env.SECRET_KEY);
  }
```